DEVELOPING_OPAL/validate/src/test/scala/org/opalj/fpcf/properties/immutability/classes/ClassImmutabilityMatcher.scala

```diff
@@ -54,15 +54,15 @@ class TransitivelyImmutableClassMatcher
 class DependentlyImmutableClassMatcher
     extends ClassImmutabilityMatcher(properties.immutability.DependentlyImmutableClass(SortedSet.empty)) {
     override def validateProperty(
-        project:    Project[_],
+        project:    Project[?],
         as:         Set[ClassType],
         entity:     scala.Any,
         a:          AnnotationLike,
         properties: Iterable[Property]
     ): Option[String] = {
         if (!properties.exists(p =>
                 p match {
-                    case DependentlyImmutableClass(latticeParameters) =>
+                    case org.opalj.br.fpcf.properties.immutability.DependentlyImmutableClass(latticeParameters) =>
```

Comment:
Do we need the entire FQN here? Specifically, in line 55 we use `properties.immutability.DependentlyImmutableClass` instead of the entire FQN